doozer/doozerlib/cli/scan_osh.py

```diff
@@ -164,7 +177,13 @@ def get_distgit_name_from_brew_nvr(self, nvr: str) -> str:
             """
         brew_package_name = self.koji_session.getBuild(buildInfo=nvr, strict=True)["package_name"]
 
-        return self.brew_distgit_mapping[brew_package_name]
+        distgit_name = self.brew_distgit_mapping.get(brew_package_name)
```

Comment:
Since we're passing a mode/type in param, better to rely on that to determine which method to call?